packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -340,6 +382,61 @@ describe(Runner, () => {
         loggerMock.verifyAll();
     });
 
+    describe('enqueue notification, send notification feature flag is enabled', () => {
+        const scanNotifyUrl = 'notify-url';
+        const notificationMessage: OnDemandNotificationRequestMessage = {
+            scanId: onDemandPageScanResult.id,
+            scanNotifyUrl: scanNotifyUrl,
+            runStatus: 'completed',
+            scanStatus: 'pass',
+        };
+
+        beforeEach(() => {
+            setupWebDriverCalls();
+
+            scannerTaskMock
+                .setup(async s => s.scan(scanMetadata.url))
+                .returns(async () => Promise.resolve(passedAxeScanResults))
+                .verifiable();
+
+            setupGenerateReportsCall(passedAxeScanResults);
+            setupSaveAllReportsCall();
+
+            serviceConfigurationMock.reset();
+            const featureFlags1: FeatureFlags = { sendNotification: true };
+            serviceConfigurationMock
+                .setup(async scm => scm.getConfigValue('featureFlags'))
+                .returns(async () => Promise.resolve(featureFlags1))
+                .verifiable(Times.once());
+        });
+
+        test.each([undefined, { scanNotifyUrl: undefined }])('Notification = %o', async notification => {
```

Comment:
change the test name to what it is testing for. like "does not send notification when url not present - %o"